### DISCUSSION TITLE
What is the replacement for MenuIcon in the package @heroicons/react/outline@v2

### DISCUSSION THREAD

**Discussion by aymardika** [tailwindlabs/heroicons#804] [Help]
Hi Team, 
I just upgraded from @heroicons/react/outline@v1 to @heroicons/react/outline@v2. I am getting this error when navigating the application home page :

**Error**: You're trying to import _**`@heroicons/react/outline/MenuIcon`**_ from Heroicons v1 but have installed Heroicons v2. Install `@heroicons/react@v1` to resolve this error.

The corresponding block of code is below :

  <XIcon className="block h-6 w-6" aria-hidden="true" />
    <MenuIcon className="block h-6 w-6" aria-hidden="true" />
  </Popover.Button>
</div>

Please could someone indicate me the replacement for MenuIcon in  @heroicons/react/outline@v2  ?
@RobinMalfait , please help!

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

You either have to keep using v1, then you won't have any errors.
Or you have to migrate your imports / names to v2 (note: this is an all new icon set, so it may or may not work visually with your current designs). More info: https://github.com/tailwindlabs/heroicons/releases/tag/v2.0.0
